test/e2e/framework.go

```diff
@@ -200,6 +214,14 @@ func (f *Framework) afterEach() {
 	f.Client = nil
 }
 
+func (f *Framework) CreateNamespace(baseName string, c *client.Client, labels map[string]string) (*api.Namespace, error) {
+	ns, err := createTestingNS(baseName, c, labels)
```

Comment:
`namespace.go` is now using `Framework`. I'd still argue that we should update it to use this function instead of `createTestingNS`, though I leave it up to you whether to punt that to another PR or do here.